source/computer/commands.rst

```diff
@@ -1,2 +1,100 @@
 常用命令
 ========
+
+Linux 下有成百上千个命令，每个命令都有众多选项。这一节介绍一些常用命令的常用用法。
+
+
+.. rubric:: 命令目录
+
+.. hlist::
+    :columns: 6
+
+    -   压缩与解压
+
+        - `tar`_
+
+    -   文件查看
+
+        - `cat`_
+        - `head`_
+        - `tail`_
+        - `less`_
+
+    -   文件处理
+
+        - `gawk`_
+        - `grep`_
+        - `wc`_
+        - `sort`_
+        - `uniq`_
+
+    -   文件搜索
+
+        - `locate`_
+        - `find`_
+
+    -   输出
+
+        - `echo`_
+        - `printf`_
```

Comment:
More thoughts:

1. Remove `echo`: I think we will mention it in Linux 101, when we talk about **PATH**
2. Remove `printf`: never used it unless writing shell scripts
3. Sort commands alphabetically?

Reply:
Yes to all.

Reply:
Done in 747ef20826b56dff80ff698bf71dbfe1b1238493.